packages/runtime/container-runtime/src/gc/gcDefinitions.ts

```diff
@@ -221,26 +222,49 @@ export const GCNodeType = {
  */
 export type GCNodeType = (typeof GCNodeType)[keyof typeof GCNodeType];
 
+/**
+ * The garbage collection sweep message.
+ * @internal
+ */
+export interface ISweepMessage {
+	type: "Sweep";
+	// The ids of nodes that are deleted.
+	contents: { deletedNodeIds: string[] };
```

Comment:
On second thought, I don't think this `contents` layer is doing any good. Just put the props directly here as siblings to `type`. Right? Why not?

Reply:
Changed it. It just makes the type of `GarbageCollectionMessage` more complex which is probably okay.